controllers/v1/observationsController.js

```diff
@@ -225,7 +226,13 @@ module.exports = class Observations extends Abstract {
 
             try {
 
-                let result = await observationsHelper.create(req.query.solutionId, req.body.data, req.userDetails, req.rspObj.userToken);
+                let result = await observationsHelper.create(
+                    req.query.solutionId, 
+                    req.body.data, 
+                    req.userDetails.id, 
+                    req.rspObj.userToken,
+                    req.query.programId
```

Comment:
@aman-tunerlabs - Older apps will not pass programId right ? We can derive the same from solution itself right ?

Reply:
@aman-tunerlabs - Remove programId from this API